var/spack/repos/builtin/packages/glib/package.py

```diff
@@ -28,6 +28,7 @@ class Glib(MesonPackage, AutotoolsPackage):
 
     license("LGPL-2.1-or-later")
 
+    version("2.79.1", sha256="b3764dd6e29b664085921dd4dd6ba2430fc19760ab6857ecfa3ebd4e8c1d114c")
```

Comment:
This is not a release version but a development version. If there isn't an urgent reason to add this, we should skip it.